### DISCUSSION TITLE
返回辦公室其實難提高生產力，專家估 2024 年有望增更多彈性

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1823] [General]
軟體公司 Atlassian 的 Team Anywhere 負責人 Annie Dean 在報告中提到，到 2024 年底，高層將承認 RTO（Return-to-Office Policy，「返回辦公室」政策）規定並沒有提高工作效率，因為關鍵一直是工作靈活性。

資料來源：https://technews.tw/2023/12/29/wfm-vs-return-to-office/

之前實習過，在家上班可以節省很多通勤時間，對於員工來說會比較好，
但對業主來說，一些機密文件也會擔心是否會被員工散佈出去，
也可能會擔心工作效率下降等，畢竟在家那麼舒服，工作到一半就想躺下來耍廢：）
對此已有研究顯示其實在公司辦公也不會提高效率，因為關鍵是工作靈活性
個人認為可以採取一週需進辦公室幾天，剩下時間員工可以在家或自由安排工作地點，來提升員工的工作熱情，不會感到乏味。

**Comment by eric202501** [comment]
確實在家辦公確實有許多優勢，如節省通勤時間、提高生活品質，但同時也面臨一些問題，如資訊安全和工作效率下降。而且公司可能擔心居家辦公導致協同合作和降低團隊凝聚力。

**Comment by yuchien-lin** [comment]
我也覺得在家上班比較方便，因為可以更好的調配時間，但面臨問題也很多，其中對於公司機密外洩，我相信是很多高科技公司無法承擔的風險，因為一但被競爭對手先做出來申請專利，那對公司的虧損是非常巨大的，所以在家上班有利有弊，公司應制定好相應的對策，才能使公司的員工有更高的生產力。
